iris_grib/message.py

```diff
@@ -20,6 +20,14 @@
 
 _SUPPORTED_GRID_DEFINITIONS = (0, 1, 5, 10, 12, 20, 30, 40, 90, 140)
 
+# Alias names for eccodes spatial computed keys.
+KEY_ALIAS = {
+    'latitude': 'latitudes',
+    'longitude': 'longitudes',
+    'latitudes': 'latitude',
+    'longitudes': 'longitude',
+}
```

Comment:
Confusingly, it appears that both the singular and plural computed keys are valid, but on for certain grids ... well, that's what the testing is telling me, and that has driven this particular change.